src/libs/isInputAutoFilled.ts

```diff
@@ -1,10 +1,11 @@
+import {TextInput} from 'react-native';
 import isSelectorSupported from './isSelectorSupported';
 
 /**
  * Check the input is auto filled or not
  */
-export default function isInputAutoFilled(input: Element): boolean {
-    if (!input?.matches) {
+export default function isInputAutoFilled(input: (TextInput | HTMLElement) | null): boolean {
```

Comment:
NAB - Is `input: (TextInput | HTMLElement) | null` not funky? I would have expected `input: (TextInput | HTMLElement | null)` to work and look better, no? I'd highly appreciate a lesson why it's necessary we do this, if anyone has the time 😅